src/main/resources/files.lst

```diff
@@ -1,14 +1,15 @@
 src/main/resources/files/tomee/index.html.tpl
 src/main/resources/files/securedURLs.ini.tpl
+src/main/resources/files/ClientController.java.tpl
 src/main/resources/files/RestApplication.java.tpl
-src/main/resources/files/test-client/MPJWTToken.java.tpl
-src/main/resources/files/test-client/liberty/JWTClient.java.tpl
-src/main/resources/files/test-client/JWTClient.java.tpl
-src/main/resources/files/test-client/privateKey.pem.tpl
-src/main/resources/files/config.yaml.tpl
 src/main/resources/files/kumuluzEE/RestApplication.java.tpl
+src/main/resources/files/kumuluzEE/config.yaml.tpl
+src/main/resources/files/kumuluzEE/service-b/RestApplication.java.tpl
+src/main/resources/files/kumuluzEE/service-b/config.yaml.tpl
+src/main/resources/files/MPJWTToken.java.tpl
+src/main/resources/files/Service.java.tpl
 src/main/resources/files/MetricController.java.tpl
-src/main/resources/files/ApplicationSecurityData.java.tpl
+src/main/resources/files/readme.md.top.tpl
 src/main/resources/files/thorntail-v2/web.xml.tpl
```

Comment:
@rdebusscher  When I select MP 2.2 and Thorntail and *only* REST, no JWT, I still get this for service-a and I think it is superfluous:

```
karm@local:/tmp$ cat th-rest-demo/demo/service-a/src/main/webapp/WEB-INF/web.xml
<?xml version="1.0" encoding="UTF-8"?>
<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
		 http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd" version="3.1">

    <security-constraint>
        <display-name>Secure All</display-name>
        <web-resource-collection>
            <web-resource-name>Secure REST Endpoint</web-resource-name>
            <url-pattern>/data/protected</url-pattern>
        </web-resource-collection>
        <auth-constraint>
            <role-name>*</role-name>
        </auth-constraint>
    </security-constraint>
```

Reply:
@Karm  Fixed in latest commit, thx!